src/openrct2/ride/Vehicle.h

```diff
@@ -152,30 +152,30 @@ enum VEHICLE_STATUS
 
 struct rct_vehicle_sound_params;
 
-enum VEHICLE_TRACK_SUBPOSITION : uint8_t
+enum class VehicleTrackSubposition : uint8_t
 {
-    VEHICLE_TRACK_SUBPOSITION_0,
+    None,
```

Comment:
`Default` as its not nothing its just what all normal rides are

Reply:
Okay, the 0 didn't give much information, I will change it.